packages/docusaurus-theme-classic/src/theme/Blog/Components/Author/GeneratedImage/index.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import clsx from 'clsx';
+import Link, {type Props as LinkProps} from '@docusaurus/Link';
+import type {Props} from '@theme/Blog/Components/Author/GeneratedImage';
+
+function MaybeLink(props: LinkProps): JSX.Element {
+  if (props.href) {
+    return <Link {...props} />;
+  }
+  return <>{props.children}</>;
+}
+
+export default function GeneratedImage({
+  name,
+  link,
+  className,
+}: Props): JSX.Element {
+  return (
+    <MaybeLink href={link} className="avatar__photo-link">
+      <svg
```

Comment:
do we really need an svg for that?

I would rather use a round div with linear gradient CSS, users would find it easier to override without swizzling